en/middleware/micrortps_manual_code_generation.md

```diff
@@ -1,187 +0,0 @@
-# Manually Generate Client and Agent Code
```

Comment:
So as noted in SUMMARY.md where I removed the link - we might need a topic like this for microDDS and microROS. IN which case a new one would be created explaining how.